packages/Maleo.js/src/build/index.ts

```diff
@@ -5,17 +5,23 @@ import webpack, { Configuration } from 'webpack';
 import { createWebpackConfig, loadUserConfig } from './webpack/webpack';
 
 import { IBuildOptions } from '@interfaces/build/IBuildOptions';
-import { Context } from '@interfaces/build/IWebpackInterfaces';
+import { Context, CustomConfig, StaticPages } from '@interfaces/build/IWebpackInterfaces';
+import { buildStatic } from '@build/static/static';
 
-export const getConfigs = (options: IBuildOptions): Configuration[] => {
+const cwd = process.cwd();
+
+export const getUserConfig = (): CustomConfig => {
```

Comment:
why do we need to split up `getUserConfig` and `getConfigs`? 
since `getConfigs` responsibility is to get all the configuration required, including user's config